pkgs/by-name/nb/nbdkit/package.nix

```diff
@@ -0,0 +1,97 @@
+{
+  stdenv,
+  testers,
+  fetchurl,
+  autoreconfHook,
+  makeWrapper,
+  pkg-config,
+  bash-completion,
+  gnutls,
+  libtool,
+  curl,
+  xz,
+  zlib-ng,
+  libssh,
+  libnbd,
+  lib,
+  cdrkit,
+  e2fsprogs,
+}:
+
+stdenv.mkDerivation (finalAttrs: {
+  pname = "nbdkit";
+  version = "1.40.4";
+
+  src = fetchurl {
+    url = "https://download.libguestfs.org/nbdkit/${lib.versions.majorMinor finalAttrs.version}-stable/nbdkit-${finalAttrs.version}.tar.gz";
+    hash = "sha256-hGoc34F7eAvHjdQHxcquNJhpwpL5CLfv2DBZKVmpcpw=";
+  };
+
+  prePatch = ''
+    # some scripts hardcore /usr/bin/env which is not available in the build env
+    patchShebangs .
+  '';
+
+  strictDeps = true;
+
+  nativeBuildInputs = [
+    bash-completion
+    autoreconfHook
+    makeWrapper
+    pkg-config
+    e2fsprogs # for Autoconf: xorriso, mkisofs
+    cdrkit # for Autoconf: mke2fs
+  ];
+
+  buildInputs = [
+    bash-completion
+    gnutls
+    libtool
+    curl
+    xz
+    zlib-ng
+    libssh
+    libnbd
+    cdrkit
+    e2fsprogs
+  ];
+
+  configureFlags = [
+    "--with-ext2"
+    "--enable-linuxdisk"
+    "--with-curl"
+    "--with-iso"
+    "--with-ssh"
+    "--with-zlib"
+    "--with-libnbd"
+    "--disable-rust"
+    "--disable-golang"
+    "--disable-perl"
+    "--disable-ocaml"
+    "--disable-tcl"
+    "--disable-lua"
+    "--without-libguestfs"
+    "--disable-example4"
+    "--disable-floppy"
+  ];
+
+  installFlags = [ "bashcompdir=$(out)/share/bash-completion/completions" ];
+
+  postInstall = ''
+    for bin in $out/bin/*; do
+      wrapProgram "$bin" \
+        --prefix PATH     : "$out/bin:${e2fsprogs}/bin:${cdrkit}/bin"
```

Comment:
* Shell indent is typically 4 chars?
* Why lots of space in "--prefix PATH [spaaaace] :"?
* Use makeBinPath? `--prefix PATH : "$out/bin:${lib.makeBinPath [ e2fsprogs cdrkit ]}"`